src/cmp.test.ts

```diff
@@ -96,24 +103,133 @@ describe('cmp', () => {
                 });
             });
 
-            it('executes advertisement callback each time consent nofication triggered', () => {
-                Cookies.get.mockReturnValue('1.54321');
+            it('executes advertisement callback each time consent nofication triggered with updated state', () => {
+                Cookies.get
+                    .mockReturnValue('1.54321') // default
+                    .mockReturnValueOnce(undefined); // first call
 
                 const myCallBack = jest.fn();
                 const expectedArguments = [
                     [
+                        {
```

Comment:
We use these 5 null, 5 true or 5 false for the IAB objects a lot. Should we create 3 objects with that easy to so it's easier to read?